spec/fixtures/description-license/DESCRIPTION

```diff
@@ -0,0 +1,2 @@
+Package: xyz
+License: MIT + file LICENSE
```

Comment:
Per CRAN policy MIT is what they call a 'license template' (https://www.r-project.org/Licenses/MIT), which states a structured format for `LICENSE` you need to follow to comply and be accepted into CRAN. Deviations from this form will be rejected. So while in principle what you say is accurate, in practice these packages are using the basic MIT license.